tools/diamond.py

```diff
@@ -26,7 +26,7 @@ class Diamond(tools.Tool):
     def __init__(self, install_methods=None):
         if not install_methods:
             install_methods = [
-                tools.CondaPackage("diamond", version=CONDA_VERSION)
+                tools.CondaPackage("diamond", env="diamond", version=CONDA_VERSION)
```

Comment:
Let's omit this, and change the diamond version in `requirements-conda.txt`, which will handle global dependency resolution on Travis.